pkg/flow/logs.go

```diff
@@ -171,47 +193,62 @@ func (srv *server) logToServer(ctx context.Context, t time.Time, msg string, a .
 
 }
 
-func (srv *server) logToNamespace(ctx context.Context, t time.Time, ns *ent.Namespace, msg string, a ...interface{}) {
+func (srv *server) tagLogToNamespace(ctx context.Context, t time.Time, tn taggedNamespace, msg string, a ...interface{}) {
 
 	defer func() {
 		_ = recover()
 	}()
 
+	srv.logQueue <- &logMessage{
+		ctx: ctx,
+		t:   t,
+		msg: fmt.Sprintf(msg, a...),
+		ns:  tn.ns(),
+		tag: tn.tags(),
+	}
+}
+
+func (srv *server) logToNamespace(ctx context.Context, t time.Time, ns *ent.Namespace, msg string, a ...interface{}) {
+
+	// defer func() {
```

Comment:
This recover serves the same purpose in `tagLogToNamespace` as it does here in `logToNamespace`. I think either both are necessary or neither is?

Reply:
Looks like this has been resolved.